resources/event-sources-controller-manager/templates/500-sources-controller.yaml

```diff
@@ -0,0 +1,51 @@
+# Copyright 2019 The Kyma Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: event-sources-controller-manager
+  namespace: kyma-system
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: event-sources-controller-manager
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: 'false'
+      labels:
+        app: event-sources-controller-manager
```

Comment:
this label is good, but i have seen some more labels on other resources. ATM i cannot link to any document stating which labels to use but these here should not be wrong:

* app: {{ .Chart.Name }}-tests
* app.kubernetes.io/name: {{ .Chart.Name }}-tests
* app.kubernetes.io/managed-by: {{ .Release.Service }}
* app.kubernetes.io/instance: {{ .Release.Name }}
* helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}

Reply:
Good suggestion

Reply:
Done. Except for `helm.sh/chart`, which we decided not to use in #5025